doc/userDocs/usersGuide.md

```diff
@@ -206,16 +230,39 @@ Handling clusters whose credential expire is discussed [below](#expire).
 When using both a driverCluster and appCluster, you can use separate kubeconfig files for each cluster,
 or use one file with multiple contexts. More information about using multiple contexts in a kubeconfig file is located at (https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/).
 
-#### NodePort and LoadBalancer Services<a name="loadbalancer"></a>
-
-The application instances can use either a NodePort or LoadBalancer service to enable incoming communication from the workload driver pods (https://kubernetes.io/docs/concepts/services-networking/service/).
-Most clusters will support at least one of these ingress methods, and some may support both.
-You will need to check the documentation for your cluster to determine which is supported.
+#### Configuring Driver to Application Communication<a name="appingressmethod"></a>
+
+The Weathervane workload driver interacts with the Weathervane application through a Kubernetes service fronting the application's web-server 
+tier. Weathervane 
+allows the user to select the type of Kubernetes service to be used.  Your selection will depend 
+on the types of services supported by your cluster and the goals of your tests.  
+
+Weathervane supports the use of 
+[LoadBalancer, NodePort, or ClusterIP services](https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types).   You 
+indicate the type of service you want to use by setting the *appIngressMethod* parameter 
+to `loadbalancer`, `nodeport`, or `clusterip`.  The default value for this parameter 
+is `loadbalancer`.
```

Comment:
Maybe add a note that for vanilla / custom created clusters (non cloud), that loadbalancer will likely not apply and this will need to be changed from the default value?

Reply:
agreed.  i will add this.

Reply:
added